common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -164,6 +164,7 @@ class SubsectionGradingPolicyA11yTest(SubsectionGradingPolicyBase):
     """
     a11y = True
 
+    @pytest.mark.skip(reason='This test fails when using the new courseware MFE.')
```

Comment:
@jmbowman & @wittjeff : Here and below are the two a11y tests I've skipped in this PR. Please let me know your thoughts about how this skipping aligns with our future a11y testing in regards to MFEs.

Reply:
Progress chart tests will be obsolete with the new Progress tab redesign.

Reply:
- When will that new Progress tab be deployed to production?
- In finishing up the Progress tab work, does the team plan to write a new a11y test for the page? Or simply remove this test?

Reply:
I believe Engage squad is gradually rolling it out to prod-edx currently. ( @jmyatt ? )

Reply:
we plan to get to 50% rollout within a week, then up to 100% by around Aug 1.  Thanks for the heads up on a11y - we'll consider whether adding a new a11y test is the right thing to do